### DISCUSSION TITLE
How to change Root Locus plot color

### DISCUSSION THREAD

**Discussion by AlHasanSony** [python-control/python-control#803] [Q&A]
The plot _color_ of the **roots** seems like similiar while using **rlocus** function. Instead of showing the default _color_, we wanted to show different color. 
So, how to change the color of the desired plot which used **rlocus**

**Comment by roryyorke** [comment]
This was already asked for in #350.

For now, you can `findobj` the lines after plotting, and then set the colours:

```python
import numpy as np
import matplotlib.pyplot as plt

import control as ct

# bit of a hack
def findlines(artist):
    return (isinstance(artist, plt.Line2D)
            and artist.get_linestyle()=='-'
            and isinstance(artist.get_xdata(), np.ndarray))

g = ct.tf([1],[1,1]) * ct.tf([1,3],[1,2]) * ct.tf([1],[1,1,16])

ct.rlocus(g);

h = plt.gca().findobj(match=findlines)

for i, h_ in enumerate(h): h_.set_color(f'C{i}')

plt.show()
```

![image](https://user-images.githubusercontent.com/110974/201528222-d4a6d4aa-3f71-4f8f-9bfb-fb62c2d1cece.png)
